include/printing_solver.h

```diff
@@ -30,11 +30,11 @@ namespace smt {
  */
 enum PrintingStyleEnum
 {
-  DEFAULT = 0,
-  BTOR_FMT,
-  CVC4_FMT,
-  MSAT_FMT,
-  YICES_FMT2
+  DEFAULT_STYLE = 0,
+  BTOR_STYLE,
```

Comment:
Quick question on this: do you actually use all these different styles? I'm wondering if we should only include the relevant ones. Otherwise users might expect different output if they pass different styles. The `BTOR_STYLE` is probably unused, right?